src/api/Feed.js

```diff
@@ -394,38 +478,77 @@ class Feed extends Base {
         this.file = file;
         this.errors = [];
         this.errorCallback = onError;
-        const annotationsPromise = shouldShowAnnotations
-            ? this.fetchAnnotations(permissions, shouldShowReplies)
+        const annotationsPromise =
+            !shouldUseUAA && shouldShowAnnotations
```

Comment:
This is really hard to read. If we don't need to make these calls at all, should we have a separate UAA-specific `feedItems` method rather than passing a `shouldUseUAA` flag?

Reply:
Not sure if it's necessary since all of these promises should be going away after `shouldUseUAA` FF is gone. I can add a comment around this to make it easier to read.